modules/plot/src/plot.cpp

```diff
@@ -182,7 +182,7 @@ namespace cv
             void render(Mat &_plotResult)
             {
                 //create the plot result
-                plotResult = Mat::zeros(plotSizeHeight, plotSizeWidth, CV_8UC3);
+                plotResult = Mat(plotSizeHeight, plotSizeWidth, CV_8UC3, plotBackgroundColor);
```

Comment:
There is no MAT constructor that takes a Scalar as an extra argument inside OpenCV...